clang/www/cxx_status.html

```diff
@@ -1133,8 +1133,8 @@ <h2 id="cxx17">C++17 implementation status</h2>
     <!-- Issaquah 2016 papers -->
     <tr>
       <td>Matching template template parameters to compatible arguments</td>
-      <td><a href="https://wg21.link/p0522r0">P0522R0</a></td>
-      <td class="partial" align="center">Partial <a href="#p0522">(10)</a></td>
+      <td><a href="https://wg21.link/p0522r0">P0522R0</a> (<a href="#dr">DR</a>)</td>
+      <td class="full" align="center">Clang 19 <a href="#p0522">(10)</a></td>
```

Comment:
This should have been "unreleased", not "full".
Sorry I missed that during review